devtools/format_md.sh

```diff
@@ -8,6 +8,7 @@ while getopts c option; do
   case "${option}" in
 
   c) op="check" ;;
+  *) ;;
```

Comment:
Ah, @uint could you install `shellcheck` on your system? It is executed automatically on all our scripts if available because of this line: `command -v shellcheck >/dev/null && shellcheck "$0"`